frontend/cypress/integration/import/import.spec.ts

```diff
@@ -113,8 +117,7 @@ describe('imports', () => {
         cy.get("[data-testid='IMPORT_CONFIGURATION_BUTTON']").click();
         // cy.contains('Import completed');
```

Comment:
I'm wondering why this commented out code was commented out. Ideally UI tests should wait on the UI changes not HTTP API interception

Reply:
However if your change fixes the issue then thumbs up

Reply:
I dont know

Reply:
Added this back now @kwasniew, good spot. Now we wait for the POST to finish and then validate ui.